tests/Nox.Reference.Data.World.Tests/CountriesInfoTests.cs

```diff
@@ -39,11 +41,14 @@ public void CountryInfo_WithIso3Alpha_ReturnsValidInfo()
         });
     }
 
-    //TODO: Play with Include
     [Test]
     public void CountryInfo_WithIso3AlphaAndTranslationForCountry_ReturnsTranslation()
     {
-        var translation = _worldDbContext.Countries.Get("ZAF")!.GetTranslation("en")!;
+        var country = _worldDbContext.Countries
```

Comment:
Why we use include here?  We should not have reference on EF here

Reply:
let's discuss this on a call